README.md

```diff
@@ -165,6 +165,69 @@ Depending on the type of the joystick and the personal preference for operation,
 
   > **Note** If for some reason the commands are not passed on to the simulation, try loading, setting and writing the correct parameters for the aircraft using QGC.
 
+Extended Wind Plugin
+-------------------------
+
+The existing wind plugin in RotorS has been extended to allow the use of a custom, static wind field for a given world. This is done by enabling the wind plugin macro in the aircraft base file (in our case `techpod_base.xacro`, which is found in `rotors_simulator/rotors_description/urdf`), and feeding it the correct parameters and files.
+
+#### Wind Plugin Macro
+Here's an example of the lines to be added in the base file, containing numerous user-defined variables:
+
+```
+<xacro:wind_plugin_macro
+  namespace="${namespace}"
+  xyz_offset="0 0 0"
+  wind_direction="1 0 0"
+  wind_force_mean="0.0"
+  wind_force_variance="0.0"
+  wind_speed_mean="5.0"
+  wind_speed_variance="0.0"
+  wind_gust_direction="0 1 0"
+  wind_gust_duration="0.0"
+  wind_gust_start="0.0"
+  wind_gust_force_mean="0.0"
+  wind_gust_force_variance="0.0"
+  custom_static_wind_field="true"
+  wind_path="$(find rotors_gazebo)/models/hemicyl/wind2.txt">
+</xacro:wind_plugin_macro>
+```
+The four values that need to be set are `wind_direction` and `wind_speed_mean`, which specify the default wind field when the aircraft is flying outside of the desired wind field, the boolean `custom_static_wind_field` which, when set to `true`, enables the extended functionality, as well as the string `wind_path` which describes the path (from `~/.ros`) to the text file specifying the wind field.
```

Comment:
Units are specified in the component_snippets.xacro file. I added this information in the documentation.